### DISCUSSION TITLE
Weight Types

### DISCUSSION THREAD

**Discussion by caniyabanci76** [cubiq/ComfyUI_IPAdapter_plus#472] [Q&A] [answered]
Is there any documentation or information about the different weight types under IPA Advanced node?  (`ease in, ease out, ease in-out` etc)   Also in the Standard Node, with the simplified weight types `standard`, `prompt is more important` etc,   are they the equivalent of the any particular ones in the Advanced node?

**Comment by cubiq** [comment, marked as answer]
standard == linear, prompt is more important == ease out

I don't have a documentation at the moment, sorry. It basically varies how the weights are applied to the unet block.
